sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java

```diff
@@ -540,16 +540,53 @@ void testTimePartitioningClustering(
     }
   }
 
-  void testTimePartitioning(BigQueryIO.Write.Method insertMethod) throws Exception {
+  void testTimePartitioningWithoutClustering(BigQueryIO.Write.Method insertMethod)
+      throws Exception {
     testTimePartitioningClustering(insertMethod, true, false);
   }
 
-  void testClustering(BigQueryIO.Write.Method insertMethod) throws Exception {
+  void testTimePartitioningWithClustering(BigQueryIO.Write.Method insertMethod) throws Exception {
     testTimePartitioningClustering(insertMethod, true, true);
   }
 
+  void testClusteringWithoutPartitioning(BigQueryIO.Write.Method insertMethod) throws Exception {
+    testTimePartitioningClustering(insertMethod, false, true);
+  }
+
+  void testNoClusteringAndNoPartitioning(BigQueryIO.Write.Method insertMethod) throws Exception {
+    testTimePartitioningClustering(insertMethod, false, false);
+  }
```

Comment:
I don't think those are very valuable. We can use `testTimePartitioningClustering` directly